README.md

```diff
@@ -27,47 +30,74 @@
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+- [❓ FAQ ](#faq)
 - [📝 License](#license)
 
-# 📖 [Hello Microverse!] <a name="about-project"></a>
-
-This is my first project with Microverse using Github professional flow
+# 📖 Hello Microverse! <a name="about-project"></a>
+This is my first project with Microverse using Github professional flow and it's a simple website that is built in html & css.
 **[Hello Microverse!]**
 
 ## 🛠 Built With <a name="built-with">HTML, CSS</a>
 
```

Comment:
- [ ] Kindly 😃 it will be better if you fix the Built With section and make it like the original REadme as you see you wrote the HTML and CSS next to the title, so please try to put them below the title inside the tech stack ul take a look below to understand 💯😊 and with that, your Readme will be professional and attract the user's attention 👌

| Professional README| Your README| 
| --- | --- |
|   ![image](https://github.com/GeekyHacks/Hello-Microverse/assets/83360636/69df1d9e-8267-48d3-a9fc-271ab250d035)   |    ![image](https://github.com/GeekyHacks/Hello-Microverse/assets/83360636/82855971-b05b-41d4-9e0f-29d97101954f)   |


| Professional README| Your README| 
| --- | --- |
|   ![image](https://github.com/GeekyHacks/Hello-Microverse/assets/83360636/75450c5b-6833-44c9-a87e-546561a7c36f)  |    ![image](https://github.com/GeekyHacks/Hello-Microverse/assets/83360636/82855971-b05b-41d4-9e0f-29d97101954f)   |


- Hint:👍 

- You can see the snippet code of this section below 😊💯

```snippet
## 🛠 Built With <a name="built-with"></a>

### Tech Stack <a name="tech-stack"></a>


<details>
  <summary>Client</summary>
  <ul>
    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started">HTML</a></li>
    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
 
  </ul>
</details>
```